common/utils/filters.ts

```diff
@@ -35,6 +36,14 @@ export const getAggregationFilterByName = (
     : [];
 };
 
+export const getAggregationContributors = (
+  aggregations: CatalogueAggregations | undefined
+): CatalogueAggregationContributorsBucket[] | [] => {
+  return aggregations && aggregations?.['contributors']?.buckets
+    ? aggregations?.['contributors']?.buckets
+    : [];
+};
```

Comment:
What's the difference between this and `getAggregationByName(aggs, 'contributors')`?

Reply:
i was having issues with the return type so i had to make its own one CatalogueAggregationContributorsBucket[]. the object type return is different.

Reply:
Yea, this is actually something I would like to address in the API - so I can use this as an example.

Makes sense to leave this then.